deepchecks/tabular/context.py

```diff
@@ -102,13 +106,13 @@ def __init__(self,
             self.predict_proba = self._predict_proba
 
     def _validate_data(self, data: pd.DataFrame):
-        # Validate only up to 10000 samples
-        data = data.sample(min(10_000, len(data)))
-        for df_features in self.features:
+        # Validate only up to 100 samples
+        data = data.sample(min(100, len(data)))
+        for feature_df in self.features_df:
```

Comment:
These names are confusing.
"features_df" represents a list of dataframe, each dataframes is of all features. so maybe should be "features_dfs" or "feature_df_list". anyway, the singular version of it (meaning, the name of an object in it) can't be "feature_df" as it's not a df of one feature.

Sorry for nirpicking, but I wasted time understanding this and so will the next person

Reply:
Agreed